src/README.md

```diff
@@ -1386,6 +1404,7 @@ static void GetUserInfo(const FunctionCallbackInfo<Value>& args) {
 [N-API]: https://nodejs.org/api/n-api.html
 [Oilpan]: https://v8.dev/blog/oilpan-library
 [Oilpan documentation in Chromium]: https://chromium.googlesource.com/v8/v8/+/main/include/cppgc/README.md
+[V8 public API documentation]: https://v8.github.io/api/head/classv8_1_1Local.html
```

Comment:
The link is specifically about `v8::Local<T>`, not just "V8 public API documentation". Can we update the link text with the following?

```suggestion
[`v8::Local<T>`]: https://v8.github.io/api/head/classv8_1_1Local.html
```